### PR TITLE
test: request modules integration tests

### PR DESCRIPTION
**Pull request by xorsal**
# 🤖 Linear

Closes OPO-658

### CODE REVIEW

File: solidity/test/integration/ContractCallRequest.t.sol [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    super.setUp();
+
+    _deposit(_accountingExtension, requester, usdc, _expectedReward);
+    _deposit(_accountingExtension, proposer, usdc, _expectedBondSize);
+
+    _contractCallRequestModule = new ContractCallRequestModule(oracle);
+    _target = new MockCallee();
+  }
+
+  function test_createRequest_finalizeEmptyResponse(bytes4 _selector, bytes calldata _data) public {
```
Comment (ashitakah) [inline solidity/test/integration/ContractCallRequest.t.sol:29]: From line 30 to 47 could be move it to the setup

Comment (ashitakah) [inline solidity/test/integration/ContractCallRequest.t.sol:29]: also as the previous pr, if I were me I would delete the params and create a selector and a data for the test

File: solidity/test/integration/ContractCallRequest.t.sol [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+pragma solidity ^0.8.19;
+
+import {
+  ContractCallRequestModule,
+  IContractCallRequestModule
+} from '../../contracts/modules/request/ContractCallRequestModule.sol';
+import {MockCallee} from '../mocks/MockCallee.sol';
+import './IntegrationBase.sol';
+
+contract Integration_ContractCallRequest is IntegrationBase {
```
Comment (ashitakah) [inline solidity/test/integration/ContractCallRequest.t.sol:11]: Also probably when you finalize the request , in the oracle the status will change, would be awesome to have asserts for this status, and expectCall to check that pay al release have been called with the correct params.

File: solidity/test/integration/HttpRequest.t.sol [hunk 3]
```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {IHttpRequestModule} from '../../contracts/modules/request/HttpRequestModule.sol';
+import './IntegrationBase.sol';
+
+contract Integration_HttpRequest is IntegrationBase {
```
Comment (ashitakah) [inline solidity/test/integration/HttpRequest.t.sol:7]: similar comments that I put in the other test

**Review by ashitakah — CHANGES_REQUESTED** [review]

**Review by 0xShaito — APPROVED** [review]

**Pull request merged**